### DISCUSSION TITLE
Toggle for the aggression timer message

### DISCUSSION THREAD

**Discussion by Christley** [runelite/runelite#15124] [Q&A] [answered]
Currently if you log into a place where you've been for a while it will have a box on the left side (default) of the screen stating: 
"Unaggressive NPC timers require calibration. Teleport far away or enter a dungeon, then run until this overlay disappears".

I realise why this message exist, but can we please add a toggle to this message? It takes up so much unnecessary space on the screen when i'm not killing anything or don't care about the borders. I know how the plugin works already so I don't need to be reminded while I'm mining or fishing how the plugin works.

So it would be awesome if you could add a toggle so i can get rid of that message forever

**Comment by Nightfirecat** [comment, marked as answer]
This was already discussed here: #15078
